docs/antrea-network-policy.md

```diff
@@ -116,32 +118,36 @@ The details for these tiers are shown below:
 Any Antrea-native policy CRD referencing a static tier in its spec will now internally
 reference the corresponding Tier resource, thus maintaining the order of enforcement.
 
-The static tier resources are created as follows in the relative order of
+The default Tiers are created as follows in the relative order of
```

Comment:
should we use "static" or "default" consistently?

Reply:
Done